.github/workflows/monodocs_build.yml

```diff
@@ -0,0 +1,52 @@
+name: Docs Build
```

Comment:
The CI is still [skipping tests](https://github.com/flyteorg/flytekit/actions/runs/7170035243) Were the changes pushed up?

```suggestion
name: Monodocs Build
```

Reply:
weird... one sec